scripts/quarantine_windows_mac.yaml

```diff
@@ -31,3 +31,27 @@
   platforms:
     - nrf9160dk_nrf9160_ns
   comment: "https://nordicsemi.atlassian.net/browse/NCSDK-26461"
+
+- scenarios:
+    - applications.matter_bridge.smp_dfu.br_ble
```

Comment:
This PR (merged) should be a remedy for Matter Bridge not fitting into the FLASH:
https://github.com/nrfconnect/sdk-nrf/pull/14447